### DISCUSSION TITLE
Pull a line on mesh

### DISCUSSION THREAD

**Discussion by flaviu22** [cnr-isti-vclab/meshlab#918] [Q&A]
Is there any feature that allow to pull straight lines on mesh (I mean, to draw a line 3D over the mesh.) ? I mean, if I have 4-5 points marked on mesh, is there any possibility to pull a line between these lines ?

**Comment by flaviu22** [comment]
I guess I can use it measure line.

**Reply by alemuntoni** [reply]
Exactly, you can use the measure tool.
